src/main/resources/META-INF/rewrite/spring-framework-60.yml

```diff
@@ -31,7 +31,16 @@ recipeList:
   - org.openrewrite.apache.httpclient5.UpgradeApacheHttpClient_5
   - org.openrewrite.java.spring.framework.HttpComponentsClientHttpRequestFactoryReadTimeout
   - org.openrewrite.java.spring.framework.MigrateResponseEntityExceptionHandlerHttpStatusToHttpStatusCode
-
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.spring.framework.MigrateResponseStatusException
```

Comment:
Indeed best to reference this `MigrateResponseStatusException` from `UpgradeSpringFramework_6_0` still, otherwise folks would need to run it explicitly before seeing those changes.